### ISSUE TITLE
[Suggestion] Don't auto join channels if nickserv auth failled

### ISSUE THREAD

**Issue by Zulgrib** [hexchat/hexchat#2712]
Hello,
It would be nice if HexChat could abort channel auto join when nick authentication fails.
NickServ authentication failing means exposing IP to channels when joining them for some servers, and it would be best avoided.

**Comment by DevNoCode** [comment]
Yeah, I always thought this is dumb when constantly reconnecting to servers every second. At least add a feature please to slow this down.
